api_tests/nodes/views/test_node_relationship_institutions.py

```diff
@@ -7,9 +8,11 @@
     NodeFactory,
 )
 from osf.utils import permissions
+from website import mails
+
 
 @pytest.mark.django_db
-class TestNodeRelationshipInstitutions:
```

Comment:
This mixin was named improper so the tests ran twice, remove `Test` prefix from the sub-class means it now runs as intended, once for each class it inherits, but not once by itself.